www/docs/getting_started.md

```diff
@@ -4,7 +4,13 @@
 
 Before you begin, we recommend installing the NPM packages for the Skip API, server, and helpers.
 
-```npm install @skipruntime/api @skipruntime/server @skipruntime/helpers```
+```npm install @skipruntime/core @skipruntime/server @skipruntime/helpers```
+
+Two versions of the runtime are available, a Wasm `@skipruntime/wasm` and a native `@skipruntime/native`.
+The Wasm runtime works with both `node` and `bun`, but is limited to Wasm's 32-bit memory address space.
+The native runtime does not have this limitation, but it is currently only available for Node and is a bit more involved to install.
+
+You can install the Wasm runtime with `npm install @skipruntime/wasm` and the native runtime by following the instructions [here](https://github.com/SkipLabs/skip/blob/main/INSTALL.md#native-skip-runtime).
```

Comment:
They will have to start from the beginning anyhow, so I guess better to not link to a later section
```suggestion
You can install the Wasm runtime with `npm install @skipruntime/wasm` and the native runtime by following the [instructions](https://github.com/SkipLabs/skip/blob/main/INSTALL.md).
```